apps/minaintyg/src/components/Layout/Breadcrumbs.tsx

```diff
@@ -0,0 +1,39 @@
+import { IDSBreadcrumbs, IDSCrumb } from '@frontend/ids-react-ts'
+import { Link, Params, useMatches } from 'react-router-dom'
+
+type Match = ReturnType<typeof useMatches>[number]
+type MatchWithCrumb = Match & {
+  handle: { crumb: (data: Params<string>) => string }
+}
+
+function hasCrumb(match: MatchWithCrumb | Match): match is MatchWithCrumb {
+  return (match as MatchWithCrumb).handle !== undefined && typeof (match as MatchWithCrumb).handle.crumb === 'function'
+}
+
+function getMatchAt(index: number, matches: MatchWithCrumb[]): { pathname: string; crumb: string } | null {
```

Comment:
Could be moved out to utils

Reply:
Agreed, this would increase the testability of this function aswell!